mapper-runtime/src/main/java/com/datastax/oss/driver/internal/mapper/entity/EntityHelperBase.java

```diff
@@ -75,4 +86,70 @@ protected void throwIfKeyspaceMissing() {
               DaoKeyspace.class.getSimpleName()));
     }
   }
+
+  public List<CqlIdentifier> findMissingColumns(
+      List<CqlIdentifier> entityColumns, Collection<ColumnMetadata> cqlColumns) {
```

Comment:
I was going to comment on the fact that the generated code boxes every identifier before invoking this method:
```java
      List<CqlIdentifier> expectedCqlPKs = new ArrayList<>();
      expectedCqlPKs.add(CqlIdentifier.fromCql("userid")); // <== LIKE THIS
      List<CqlIdentifier> missingTablePksNames = findMissingColumns(expectedCqlPKs, tableMetadata.get().getPartitionKey());
```
But that's actually the right way to do it. These identifiers may be provided directly by the user with a `CqlName` annotation, so comparing them directly to `cqlColumn.asCql(...)` would not be reliable.

So nothing to change here 👍